components/log-viewer-webui/client/src/pages/SearchPage/SearchResults/SearchResultsTable/typings.tsx

```diff
@@ -24,9 +24,17 @@ interface SearchResult {
 const searchResultsTableColumns: NonNullable<TableProps<SearchResult>["columns"]> = [
     {
         dataIndex: "timestamp",
+        defaultSortOrder: "ascend",
         key: "timestamp",
         render: (timestamp: number) => dayjs(timestamp).format(DATETIME_FORMAT_TEMPLATE),
-        sorter: true,
+        sorter: (a, b) => a.timestamp - b.timestamp,
+
+        // Specifying a third sort direction removes ability for user to cancel sorting.
+        sortDirections: [
+            "ascend",
+            "descend",
+            "ascend",
+        ],
         title: "Timestamp",
         width: 15,
     },
```

Comment:
(optional: utterly nit-picking) since there're many fields already, i'd probably group the fields:
```suggestion
    {
        dataIndex: "timestamp",
        key: "timestamp",

        title: "Timestamp",
        render: (timestamp: number) => dayjs(timestamp).format(DATETIME_FORMAT_TEMPLATE),
        width: 15,
        
        // Specifying a third sort direction removes ability for user to cancel sorting.
        defaultSortOrder: "ascend",
        sortDirections: [
            "ascend",
            "descend",
            "ascend",
        ],
        sorter: (a, b) => a.timestamp - b.timestamp,
    },
```

Reply:
I would be happy to do this, but the linter will complain about sort keys